tests/plugins/test_koji_promote.py

```diff
@@ -961,13 +961,15 @@ def test_koji_promote_image_tags(self, tmpdir, os_env, additional_tags):
          None),
 
     ])
-    @pytest.mark.parametrize('has_config', [
-        True,
-        False,
+    @pytest.mark.parametrize(('has_config', 'is_autorebuild'), [
+        (True,
```

Comment:
as it works now is it provides, true, true & false, false now
as these 2 test cases are independent I don't need full set t,t & t,f & f,f & f,t

I could even use has_config for is_autorebuild, but I wanted to explicitly name it